### ISSUE TITLE
Single Player Phantom Objects from Previous Levels

### ISSUE THREAD

**Issue by ctralie** [JACoders/OpenJK#974]
Reporting a bug? Please make sure you've given the following information - thanks!

**Operating system and version:**
Ubuntu 16.04.
v1.0.1.1 linux-x86_64

**Is this for single player or multiplayer?**
Single player

**Description of the bug (and if possible, steps to reproduce the bug):**
I've noticed a hilarious bug that doesn't affect gameplay, but which is very unusual.  Every once in a while, there will be random objects from the scenery from two levels ago.  These objects don't have any collision detection, so I can go right through them.  But they sometimes block things from view.  I'm attaching some of the more awesome examples (mostly from the desert level, superimposing objects from the first level outside of the academy, which was quite a juxtaposition)

![screenshot from 2018-04-02 20-42-11](https://user-images.githubusercontent.com/1270246/39598370-55fa7246-4ee6-11e8-98db-4e9377a2068b.png)
![screenshot from 2018-04-02 20-42-32](https://user-images.githubusercontent.com/1270246/39598371-5608116c-4ee6-11e8-9565-346606191f1e.png)
![screenshot from 2018-04-02 20-44-05](https://user-images.githubusercontent.com/1270246/39598372-5611e246-4ee6-11e8-8891-a72d9af396b9.png)
![screenshot from 2018-04-02 20-45-37](https://user-images.githubusercontent.com/1270246/39598373-561d71ba-4ee6-11e8-9563-725425916303.png)
![screenshot from 2018-04-02 20-48-54](https://user-images.githubusercontent.com/1270246/39598374-562945d0-4ee6-11e8-8fe4-20ef8b712d0f.png)
![screenshot from 2018-04-09 13-00-07](https://user-images.githubusercontent.com/1270246/39598375-56359bdc-4ee6-11e8-8091-2b9e5e02ab31.png)

**Comment by ensiform** [comment]
Are you able to build from git to see if the cmake changes fixes this?

**Comment by ctralie** [comment]
Yeah I can try to do that today.  You want me to try 56bccd9 on master?

**Comment by ensiform** [comment]
Sure

**Comment by ctralie** [comment]
My sincerest apologies for the delay on this.  To assess this, I'm going to
have to play through the levels, and I just haven't had the time!  I know
you probably want to get the issue count down, so feel free to close it if
you want (definitely not high priority).  I mainly wanted to share these
hilarious screenshots and alert you to the issue.

Or, you can wait, and I'll do it sometime within the month.
Thanks,
Chris

**Comment by sotpapathe** [comment]
I encountered this issue when playing with the binary downloaded automatically by Lutris which is `openjk-2018-02-26-e3f22070-linux-64.tar.gz` I did not see any issues when playing with the latest commit compiled from source (currently 52030235f052772008d99e6ccb16de48e7ddb688).

In my case the trees on Tatooine were properly placed on the ground, for a moment I thought they were part of the level :stuck_out_tongue: Definitely made it more interesting.

**Comment by ctralie** [comment]
Very interesting!  I actually really enjoyed it, and it definitely didn't
impede gameplay much; it seems to just be a rendering error, not an error
with any physics
